testing/jormungandr-integration-tests/Cargo.toml

```diff
@@ -5,10 +5,12 @@ authors = [ "dev@iohk.io" ]
 edition = "2018"
 
 [dependencies]
-grpc            = "0.6.1"
-protobuf        = "=2.8.2"
+tonic = "0.2"
+prost = "0.6"
+tokio = { version = "0.2", features = ["macros"] }
 futures         = "~0.1"
```

Comment:
Burn it all down, nothing else requires the old futures. Then you can use just `futures = 0.3` as a dependency, no need to fiddle with the subcrates.